#Configuation for robot odometry EKF
#
jack_disthandle_subT: "/gtec/toa/ranging"
jack_disthandle_subA: "/gtec/toa/anchors"
jack_disthandle_pubDist: "/DT1/DT"
jack_disthandle_pubGrad: "/DT1/gradient"
jackAPIName: "DT1API"
jack_disthandle_tagID: 1
NanchorMesh: 4
GradientFlag: 0
UWBrate: 5